js/src/creatives/mute.js

```diff
@@ -0,0 +1,77 @@
+import FW from '../fw/fw';
+import VASTPLAYER from '../players/vast-player';
+import CONTENTPLAYER from '../players/content-player';
+import VPAID from '../players/vpaid';
+
+const MUTE = {};
+
+const _setMute = function (muted) {
+
+  if (typeof muted !== 'boolean') {
+    return;
+  }
```

Comment:
I think this is a job of Type checker, which pure JS does not have :) 
in this case, since you are calling this function from your own code with hardcoded values (true, false) below, I'd remove this check maybe, wdyt?

Reply:
Yup, LGTM 👍